packages/analytics-js/__fixtures__/msw.handlers.ts

```diff
@@ -77,6 +77,14 @@ const handlers = [
       },
     });
   }),
+  http.post(`${dummyDataplaneHost}/setCookie`, () => {
+    return new HttpResponse(null, {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json; charset=utf-8',
```

Comment:
Let's also set the cookies headers to simulate the actual feature.

Reply:
Addressed